src/demangle.cc

```diff
@@ -1351,6 +1351,9 @@ bool Demangle(const char* mangled, char* out, size_t out_size) {
   }
 
   std::copy_n(unmangled.get(), std::min(n, out_size), out);
```

Comment:
I suggest storing the `std::min` result in a variable and writing the null character unconditionally.

Reply:
ok